app/javascript/components/tpi/charts/cp-matrix/cp-matrix-table.jsx

```diff
@@ -0,0 +1,133 @@
+/* eslint-disable function-paren-newline */
+/* eslint-disable implicit-arrow-linebreak */
+import React from 'react';
+import PropTypes from 'prop-types';
+import Table from 'rc-table';
+import ReactTooltip from 'react-tooltip';
+import cx from 'classnames';
+import chevronIcon from 'images/icon_chevron_dark/chevron_down_black-1.svg';
+
+const COLORS = {
```

Comment:
List of available names can be seen at https://github.com/Vizzuality/laws_and_pathways/blob/develop/app/models/cp/alignment.rb#L27. Based on this list, `Not aligned` should be `Not Aligned`. I will also add `Not assessable using TPI’s methodology` on BE side to this list.

Reply:
I have covered this with my commit ;)